src/views/BED/BED.tsx

```diff
@@ -30,7 +32,7 @@ const BedProductPage = (props: { title: string }) => {
     token: token,
     components: components,
     balance: bedBalance,
-    currentSupply: bedTotalSupply,
+    currentSupply: bedTotalSupply ?? bedTotalSupplyFromBalances,
```

Comment:
here we fallback to the existing supply fetched from `useBalances()` in case the request to set.js fails

`??` is the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)

Reply:
sweet, lets use this more @0xModene